deploy/helm/grafana-operator/templates/deployment.yaml

```diff
@@ -42,6 +42,10 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
+            {{- if .Values.env.grafanaImage }}
+            - name: RELATED_IMAGE_GRAFANA
```

Comment:
I believe this is not possible, because operator-sdk looks for a "RELATED_IMAGE_" prefix, [relatedimages.go](https://github.com/operator-framework/operator-sdk/blob/master/internal/cmd/operator-sdk/generate/internal/relatedimages.go#L56)

Reply:
Sigh, I disslike OLM more and more for each time I try to understand its config.
Thanks for the link.